christfellowshipchurch/web-app-v2
# Add meetingType to group filters and search

This PR adds the `meetingType` to groups so users can select either 'In Person' or 'Online' as a filter. This attribute is also added to the Algolia config to be searched by and set up as a facet.

**TO TEST**
Play around with the search and the filters for meetingType and make sure everything is functioning as excepted. For example, I found a bug when I had my Campus set to 'Online' and the meetingType set to 'In Person'. It caused a situation where you would never get any results.

![image](https://user-images.githubusercontent.com/2528817/109053698-9dd9ac80-76a2-11eb-9a90-4fb93529c3a2.png)
![image](https://user-images.githubusercontent.com/2528817/109053772-b5b13080-76a2-11eb-87cf-bafe6eeb1812.png)
![image](https://user-images.githubusercontent.com/2528817/109180653-b3a5ab00-7750-11eb-90ba-fcdf96a44364.png)
